internal/test/issues/issue-52/issue.gen.go

```diff
@@ -116,10 +116,12 @@ type Client struct {
 	RequestEditor RequestEditorFn
 }
 
+var _ ClientInterface = &Client{}
```

Comment:
This will make sure that the `Client` struct will always implement the `ClientInterface` otherwise it will be a build failure